pkg/network/common/cniserver/cniserver.go

```diff
@@ -293,6 +295,12 @@ func (s *CNIServer) handleCNIRequest(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, fmt.Sprintf("%v", err), http.StatusBadRequest)
 		return
 	}
+	if req.Command == CNI_ADD {
```

Comment:
@pperiyasamy I would set the start time for *all* CNI operations and then at the end of the function use the req.Command to determine which metric to set. We should have both ADD & DEL metrics for e2e latency.

Reply:
sure @dcbw, now adding metric labels for both delete as well as update (though update doesn't happen from CNI).